packages/js-recommendations/README.md

```diff
@@ -0,0 +1,287 @@
+# `@algolia/js-recommendations`
+
+JavaScript package for [Algolia Recommend](https://www.algolia.com/doc/guides/algolia-ai/recommend/).
+
+## Installation
+
+```sh
+yarn add @algolia/js-recommendations@experimental
+# or
+npm install @algolia/js-recommendations@experimental
+```
+
+## API
+
+### `relatedProducts`
+
+Function to render related products.
+
+#### Usage
+
+To get started, you need a container for your results to go in. If you don’t have one already, you can insert one into your markup:
+
+```html
+<div id="relatedProducts"></div>
+```
+
+Then, inject results into it by calling the `relatedProducts` function and providing the [`container`](#container). It can be a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors) or an [Element](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement).
+
+```js
+/** @jsx h */
+
+import { h } from 'preact';
+import { relatedProducts } from '@algolia/js-recommendations';
+import algoliasearch from 'algoliasearch';
+
+const appId = 'HYDY1KWTWB';
+const apiKey = '28cf6d38411215e2eef188e635216508';
+const indexName = 'gstar_demo_test';
+
+const searchClient = algoliasearch(appId, apiKey);
+
+function renderRelatedProducts({ currentObjectID }) {
```

Comment:
We can remove this function abstraction from the code snippet.

Reply:
Removed abstraction in https://github.com/algolia/ui-components/pull/18/commits/1d53cbfc83fc5f8e4e981774917f08143849bb5a